lib/workers/repository/model/commit-message.ts

```diff
@@ -44,6 +44,27 @@ export class CommitMessage {
     return [prefix, message].join(' ').trim();
   }
 
+  public static formatCasing(message: string): string {
+    let commitMessage: any = message;
+    const hasSemiColon: boolean = commitMessage.includes(':');
+    commitMessage = commitMessage.split(':');
+    commitMessage = commitMessage.map((msg: string) =>
+      this.lowerCaseFirstWord(msg)
+    );
+    return hasSemiColon ? commitMessage.join(': ') : commitMessage.join('');
+  }
+
+  static lowerCaseFirstWord(message: string): string {
+    let copyMsg: any = message;
+    if (!copyMsg) {
+      return copyMsg;
+    }
+    copyMsg = copyMsg.trim().split(' ');
+    copyMsg[0] = copyMsg[0].toLowerCase();
+    copyMsg = copyMsg.join(' ');
+    return copyMsg;
+  }
```

Comment:
Please, consider implementing new class

```
export class SemanticCommitMessage {
  readonly type: string;
  readonly scope?: string;
  readonly description: string;

  constructor(type: string, description: string, scope?: string) {
    this.type = type;
    this.description = description;
    this.scope = scope;
  }

  format(): string {
    // do something
  }
}
```

Reply:
@RahulGautamSingh Do you mind if @pret-a-porter takes this one over?

Reply:
Nope, no worries.